Source/Core/Core/NetPlayClient.h

```diff
@@ -252,6 +252,8 @@ class NetPlayClient : public TraversalClientClient
   void DisplayPlayersPing();
   u32 GetPlayersMaxPing() const;
 
+  bool WaitForWiimoteBuffer(int _number);
```

Comment:
`const`?

Reply:
`m_wii_pad_event.Wait()` is not const.

Reply:
Gah, somehow I glossed over that.  Sorry!